tests/decorators/tests.py

```diff
@@ -291,6 +313,44 @@ def method(self):
 
         self.assertTrue(Test().method())
 
+    def test_iterable_of_decorators(self):
+        """
+        @method_decorator can accept an iterable of decorators.
+        """
+        def add_question_mark(func):
+            def _wrapper(*args, **kwargs):
+                return func(*args, **kwargs) + "?"
+            return _wrapper
+
+        def add_exclamation_mark(func):
+            def _wrapper(*args, **kwargs):
+                return func(*args, **kwargs) + "!"
+            return _wrapper
+
+        # the order should be coherent with the
+        # classical criterion of decorators application
+        # >>> @add_exclamation_mark
+        # >>> @add_question_mark
+        # >>> def func():
+        # ...    return ""
+        decorators = (
```

Comment:
At this point why not just use decorator[::-1] it will raise a TypeError exception anyway if decorator does not have a `__getitem__` attribute.